mLRS/Common/esp-lib/esp-uart-template.h

```diff
@@ -59,8 +59,8 @@ typedef enum {
   #if (UART$_TXBUFSIZE > 0) && (UART$_TXBUFSIZE < 256)
     #error UART$_TXBUFSIZE must be 0 or >= 256
   #endif
-  #if (UART$_RXBUFSIZE < 256)
-    #error UART$_RXBUFSIZE must be >= 256
+  #if (UART_RXBUFSIZE > 0) && (UART_RXBUFSIZE < 256)
```

Comment:
in the template you must wirte UART_$_xxxx
the other files are then generated from the tempate 
they are now incorrect becasue of the missing $